docker/README.md

```diff
@@ -0,0 +1,19 @@
+Building the image
+------------------
+
+First clone augeas repository then, build the image:
+
+```shell
+git clone https://github.com/hercules-team/augeas.git
+cd augeas
+docker build -t augeas -f docker/Dockerfile .
+```
```

Comment:
Not sure this section is very useful as this is a very standard way of building a Docker image.

Reply:
It says from what folder you must run `docker build` for docker to find the files it needs.